src/Reflector.php

```diff
@@ -34,4 +39,16 @@ public function isClassInternal(string $name): bool
             return false;
         }
     }
+
+    public function isFunctionInternal(string $name): bool
+    {
+        try {
+            return (new ReflectionFunction($name))->isInternal();
+
+            // TODO: leverage Better Reflection instead
```

Comment:
@Ocramius could you double check that? I may have misunderstood how it works but from what I see the identifier is correctly built but `PhpInternalSourceLocator` just tries to get the class from that identifier (which fails here as it's a function). 

See [source](https://github.com/Roave/BetterReflection/blob/master/src/SourceLocator/Type/PhpInternalSourceLocator.php#L42)

Reply:
Aye, that looks like a bug in BR